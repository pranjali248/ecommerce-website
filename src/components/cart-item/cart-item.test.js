import { shallow } from "enzyme";
import React from "react";
import CartItem from "./cart-item.component";

describe("Cart component testing", () => {
  const item = {
    name: "test",
    imageUrl: "test",
    price: 0,
    quantity: 1,
  };
  it("Shallow snapshot testing of cart-item component", () => {
    expect(shallow(<CartItem item={item} />)).toMatchSnapshot();
  });
});
