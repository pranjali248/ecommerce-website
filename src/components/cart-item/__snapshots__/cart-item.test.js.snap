// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart component testing Shallow snapshot testing of cart-item component 1`] = `ShallowWrapper {}`;
